scripts/common/install-latest-librdkafka.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
Why not just do
```
#!/bin/bash -e
```
instead of all these `|| exit 2` lines?

Reply:
Sounds good. Will do.